src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx

```diff
@@ -18,34 +18,32 @@ import React from 'react';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
 import find from 'lodash/find';
+import { push } from 'connected-react-router';
 
+import Tooltip from 'src/shared/components/tooltip/Tooltip';
+import InlineSVG from 'src/shared/components/inline-svg/InlineSvg';
```

Comment:
Why did you move these two imports? We usually try to group imports by their type; and now these two, which are components, are sitting together with helpres and hooks.

Reply:
I've put it back. I'm still getting confused with the order. Can you please remind me the correct order once again?

Reply:
I'm usually doing it like this:

1) libraries from node modules
2) non-components (redux actions or selectors, helpers, hooks), tend to group them by type
3) react components
4) typescript types (if not imported together with imports in one of the previous groups)
5) styles and, on occasion other static assets, such as images (not the svgs that we import as React components though; those go into the react components group).

What are your conventions?

Reply:
Thanks. I think I didn't separate non-component and react component and kept mixing them up. It's all clear now.